blocks/v2-footer/v2-footer.js

```diff
@@ -0,0 +1,207 @@
+import { readBlockConfig, decorateIcons, loadBlocks } from '../../scripts/lib-franklin.js';
+import { createElement, getTextLabel } from '../../scripts/scripts.js';
+
+const PLACEHOLDERS = {
+  visit: getTextLabel('visit aria label'),
+  social: getTextLabel('social aria label'),
+  channel: getTextLabel('channel aria label'),
+};
+
+/**
+ * loads and decorates the footer
+ * @param {Element} block The header block element
+ */
+
+export default async function decorate(block) {
+  const cfg = readBlockConfig(block);
+  block.textContent = '';
+
+  const footerPath = cfg.footer || '/drafts/avinash/footer';
+  const resp = await fetch(`${footerPath}.plain.html`);
+  const html = await resp.text();
+  const footer = createElement('div', 'v2-footer-container');
+  footer.innerHTML = html;
+
+  // // adding the 'block' class and 'data-block-name'
+  // // needed to load blocks
+  const eloquaFormEl = footer.querySelector('.eloqua-form');
+  eloquaFormEl?.classList.add('block');
+  eloquaFormEl?.setAttribute('data-block-name', 'eloqua-form');
+
+  // transforming icons into font-awesome
+  footer.querySelectorAll('span.icon').forEach((icon) => {
```

Comment:
We’ll replace FontAwesome gradually in the project for SVG icons. Could you make sure we’ll have an SVG icon for all social media? Make sure to implement the [latest one for Twitter](https://about.twitter.com/en/who-we-are/brand-toolkit).